script.js

```diff
@@ -0,0 +1,521 @@
+    ['1', '!', '1', '!', 'Digit1'],
+    ['2', '\x22', '2', '@', 'Digit2'],
+    ['3', '№', '3', '#', 'Digit3'],
+    ['4', ';', '4', '$', 'Digit4'],
+    ['5', '%', '5', '%', 'Digit5'],
+    ['6', ',', '6', '^', 'Digit6'],
+    ['7', '.', '7', '&', 'Digit7'],
+    ['8', '*', '8', '*', 'Digit8'],
+    ['9', '(', '9', '(', 'Digit9'],
+    ['0', ')', '0', ')', 'Digit0'],
+    ['-', '_', '-', '_', 'Minus'],
+    ['=', '+', '=', '+', 'Equal'],
+    ['backspace', 'backspace', 'backspace', 'backspace', 'Backspace'],
+  ],
+  [
+    ['tab', 'tab', 'tab', 'tab', 'Tab'],
+    ['й', 'Й', 'q', 'Q', 'KeyQ'],
+    ['ц', 'Ц', 'w', 'W', 'KeyW'],
+    ['у', 'У', 'e', 'E', 'KeyE'],
+    ['к', 'К', 'r', 'R', 'KeyR'],
+    ['е', 'Е', 't', 'T', 'KeyT'],
+    ['н', 'Н', 'y', 'Y', 'KeyY'],
+    ['г', 'Г', 'u', 'U', 'KeyU'],
+    ['ш', 'Ш', 'i', 'I', 'KeyI'],
+    ['щ', 'Щ', 'o', 'O', 'KeyO'],
+    ['з', 'З', 'p', 'P', 'KeyP'],
+    ['х', 'Х', '[', '{', 'BracketLeft'],
+    ['ъ', 'Ъ', ']', '}', 'BracketRight'],
+    ['ё', 'Ё', '\x5c', '|', 'Backslash'],
+    ['delete', 'delete', 'delete', 'delete', 'Delete'],
+  ],
+  [
+    ['caps lock', 'caps lock', 'caps lock', 'caps lock', 'CapsLock'],
+    ['ф', 'Ф', 'a', 'A', 'KeyA'],
+    ['ы', 'Ы', 's', 'S', 'KeyS'],
+    ['в', 'В', 'd', 'D', 'KeyD'],
+    ['а', 'А', 'f', 'F', 'KeyF'],
+    ['п', 'П', 'g', 'G', 'KeyG'],
+    ['р', 'Р', 'h', 'H', 'KeyH'],
+    ['о', 'О', 'j', 'J', 'KeyJ'],
+    ['л', 'Л', 'k', 'K', 'KeyK'],
+    ['д', 'Д', 'l', 'L', 'KeyL'],
+    ['ж', 'Ж', ';', ':', 'Semicolon'],
+    ['э', 'Э', '\'', '\x22', 'Quote'],
+    ['return', 'return', 'return', 'return', 'Enter'],
+  ],
+  [
+    ['shift', 'shift', 'shift', 'shift', 'ShiftLeft'],
+    ['я', 'Я', 'z', 'Z', 'KeyZ'],
+    ['ч', 'Ч', 'x', 'X', 'KeyX'],
+    ['с', 'С', 'c', 'C', 'KeyC'],
+    ['м', 'М', 'v', 'V', 'KeyV'],
+    ['и', 'И', 'b', 'B', 'KeyB'],
+    ['т', 'Т', 'n', 'N', 'KeyN'],
+    ['ь', 'Ь', 'm', 'M', 'KeyM'],
+    ['б', 'Б', '<', ',', 'Comma'],
+    ['ю', 'Ю', '>', '.', 'Period'],
+    ['/', '?', '/', '?', 'Slash'],
+    ['shift', 'shift', 'shift', 'shift', 'ShiftRight'],
+  ],
+  [
+    ['control', 'control', 'control', 'control', 'ControlLeft'],
+    ['option', 'option', 'option', 'option', 'AltLeft'],
+    ['command', 'command', 'command', 'command', 'MetaLeft'],
+    ['\x20', '\x20', '\x20', '\x20', 'Space'],
+    ['command', 'command', 'command', 'command', 'MetaRight'],
+    ['option', 'option', 'option', 'option', 'AltRight'],
+    ['&#9666;', '&#9666;', '&#9666;', '&#9666;', 'ArrowLeft'],
+    ['&#9652;', '&#9652;', '&#9652;', '&#9652;', 'ArrowUp'],
+    ['&#9662;', '&#9662;', '&#9662;', '&#9662;', 'ArrowDown'],
+    ['&#9656;', '&#9656;', '&#9656;', '&#9656;', 'ArrowRight'],
+  ],
+];
+
+const functionalKeys = ['Backspace', 'Tab', 'Delete', 'CapsLock', 'Enter', 'ShiftLeft', 'ShiftRight', 'ControlLeft', 'AltLeft', 'MetaLeft', 'MetaRight', 'AltRight', 'ArrowLeft', 'ArrowRight', 'ArrowDown', 'ArrowUp', 'Space'];
```

Comment:
as a suggestion: the array with many items may be written in a column for readability